client/extensions/woocommerce/state/data-layer/products/test/index.js

```diff
@@ -32,12 +35,28 @@ describe( 'handlers', () => {
 					type: WOOCOMMERCE_API_REQUEST,
 					method: 'post',
 					siteId: 123,
-					onSuccessAction: successAction,
+					onSuccessAction: productUpdated( 123, null, successAction ),
 					onFailureAction: failureAction,
 					body: { name: 'Product #1', type: 'simple' },
 				} )
 			);
 		} );
 	} );
+
+	describe( '#handleProductUpdated', () => {
+		it( 'should dispatch a completion action', () => {
+			const store = {
+				dispatch: spy(),
+			};
+
+			const product1 = { id: 101, name: 'Newly Created Product', type: 'simple' };
+			const completionAction = { type: '%%complete%%' };
```

Comment:
Is this just a placeholder action, or should we be importing another action-type constant?